src/main/java/com/github/havardh/javaflow/plugins/JavaflowMojo.java

```diff
@@ -120,4 +126,25 @@ private void run(Api api) {
       getLog().error(e);
     }
   }
+
+  private List<Verifier> getAllVerifiers(Map<String, Boolean> verifiersConfig, TypeMap typeMap) {
+    List<Verifier> verifiers = new ArrayList<>();
+    verifiers.add(new MemberFieldsPresentVerifier(typeMap));
+    verifiers.addAll(getOptionalVerifiers(verifiersConfig));
+    return verifiers;
+  }
+
+  private List<Verifier> getOptionalVerifiers(Map<String, Boolean> verifiersConfig) {
+    if (verifiersConfig == null) {
```

Comment:
this could be removed if you implement my proposal about default values in the Api file.